src/mpl/src/gpu/mpl_gpu_fallback.c

```diff
@@ -5,6 +5,11 @@
 
 #include "mpl.h"
 
+int MPL_gpu_get_dev_count(int *dev_cnt, int *dev_id)
+{
+    return MPL_SUCCESS;
```

Comment:
Should we still set the output variables here? I think it would be better than relying on the caller to initialize them and then check if they were updated.

Reply:
I think the caller already sets `dev_cnt` and `dev_id` as -1 and checks whether they are changed after calling `MPL_gpu_get_dev_count`. The code is in the second commit `MPIDI_GPU_mpi_init_hook`. Can you check whether it is what you want?

Reply:
Fixed